amp/parse.go

```diff
@@ -32,6 +41,62 @@ type Size struct {
 	Width          int64
 }
 
+// Policy consent types
+const (
+	ConsentDenied = iota
+	ConsentTCF1
+	ConsentTCF2
+	ConsentUSPrivacy
```

Comment:
Please set the values directly instead of relying on `iota` because this map is defined by an external source and `iota` implies the values have no specific meaning. You may want to include in the comment that these values are specified externally from the AMP project.

Reply:
Makes sense. Done